meta-resin-common/recipes-connectivity/networkmanager/networkmanager_1.14.4.bb

```diff
@@ -5,16 +5,13 @@ SECTION = "net/misc"
 LICENSE = "GPLv2+"
 LIC_FILES_CHKSUM = "file://COPYING;md5=cbbffd568227ada506640fe950a4823b \
                     file://libnm-util/COPYING;md5=1c4fa765d6eb3cd2fbd84344a1b816cd \
-                    file://docs/api/html/license.html;md5=ac20f1edc24f72480a1106871e9fbe9a \
+                    file://docs/api/html/license.html;md5=2d56a1b0c42e388aa86aef59b154e8c3 \
 "
 
 DEPENDS = " \
     intltool-native \
     libxslt-native \
     libnl \
-    dbus \
-    dbus-glib \
-    dbus-glib-native \
```

Comment:
Were these added by us or inherited from the recipe? Should these be added in the bbappend?

Reply:
Git log? :)